lib/suma/processor.rb

```diff
@@ -12,10 +12,25 @@
 module Suma
   class Processor
     class << self
-      def run(metanorma_yaml_path:, schemas_all_path:, compile:,
-output_directory: "_site")
-        Utils.log "Current directory: #{Dir.getwd}"
```

Comment:
You got rid of this line, but TBH, it won't be missed

Reply:
merged it with "Writting ..." log